### ISSUE TITLE
Allow unary "!" method call syntax

### ISSUE THREAD

**Issue by Sija** [crystal-lang/crystal#8765]
Other unary operators are parsed without a problem, yet not this one:

```cr
foo = true
foo.!
```

```
expecting any of these tokens: IDENT, CONST, +, -, *, /, //, %, |, &, ^, ~, **, <<, <, <=, ==, !=, =~, !~, >>, >, >=, <=>, ===, [], []=, []?, [, &+, &-, &*, &** (not '!')
```

Seems like a bug — and a valuable _trick_ to use, while filtering for instance (`elements.select(&.visible?.!))` 😼

**Comment by bew** [comment]
Did you test on master? I remember a PR fixed that not long ago

**Comment by straight-shoota** [comment]
This should've been fixed in #8445 and was already released in 0.32.0.

What's your Crystal version?

**Comment by Sija** [comment]
My bad, it seems like some `icr` specific thing.
Can be closed, or... (should it raise such an error in any case?)
